packages/contracts/contracts/registry/DAORegistry.sol

```diff
@@ -57,6 +60,10 @@ contract DAORegistry is InterfaceBasedRegistry {
             revert EmptyDaoName();
         }
 
+        if (!_checkNameValidity(_name)) {
```

Comment:
`_isSubdomainValid()`  instead?

If it's `_checkNameValidity()` it should return nothing and just revert when not valid

Reply:
Also, please let's take the opportunity to rename `_name` into `_daoSubdomain` (or similar). The DAO's `name` is stored elsewhere, not on-chain

Reply:
+1

Reply:
I renamed it from `_name` to `subdomain` because we move it to a free function and thus the name could be either for a plugin or a dao